uint32[2]    robot_pos_1
uint32[2]    robot_pos_2
uint32[2]    robot_pos_3
uint32[2]    robot_pos_4
uint32[2]    robot_pos_5